app/views/angular_template/test.html

```diff
@@ -0,0 +1,2 @@
+<!-- Please do not remove, will break tests :) -->
+It works!
```

Comment:
That'd work but ..chicken and egg :). I'm expecting @zeari to use it once it's merged but until it is and his code is, we have no real files to test. So I can either remove it for now, or leave it as it is and change to load real files once we have some.